modin/pandas/series.py

```diff
@@ -144,19 +147,32 @@ def __init__(
                 name = MODIN_UNNAMED_SERIES_LABEL
                 if isinstance(data, pandas.Series) and data.name is not None:
                     name = data.name
-
-            query_compiler = from_pandas(
-                pandas.DataFrame(
-                    pandas.Series(
-                        data=data,
-                        index=index,
-                        dtype=dtype,
-                        name=name,
-                        copy=copy,
-                        fastpath=fastpath,
+            if UsePlainPandasQueryCompiler.get():
+                query_compiler = PlainPandasQueryCompiler(
+                    pandas.DataFrame(
+                        pandas.Series(
+                            data=data,
+                            index=index,
+                            dtype=dtype,
+                            name=name,
+                            copy=copy,
+                            fastpath=fastpath,
+                        )
                     )
                 )
-            )._query_compiler
+            else:
+                query_compiler = from_pandas(
+                    pandas.DataFrame(
+                        pandas.Series(
+                            data=data,
+                            index=index,
+                            dtype=dtype,
+                            name=name,
+                            copy=copy,
+                            fastpath=fastpath,
+                        )
+                    )
+                )._query_compiler
```

Comment:
This should be reverted because we already create a qc by calling from_pandas.

Reply:
Done